frontend/pages/faq.js

```diff
@@ -90,7 +90,9 @@ export default function Faq({ questionsBySection, questions }) {
   // The first section should be the initial tab value
   const [searchValue, setSearchValue] = React.useState("");
 
-  const handleSearchBarChange = (type, value) => setSearchValue(value);
+  const handleSearchBarChange = (event) => {
+    setSearchValue(event.target.value);
```

Comment:
I _believe_ this won't be necessary in most cases, but WDYT_ about being oversafe around `ReferenceError` and using [optional chaining](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining) here? Chris and I have been nerding out about it being our newest favorite JS feature 😄: 
```suggestion
    setSearchValue(event?.target?.value);
```

Reply:
SGTM :)